pages/phases/assessments/training.md

```diff
@@ -6,7 +6,7 @@ sidenav: hiring-phases
 phase: assessments
 sub-phase-order: 015
 title: Training SMEs to Conduct Assessments
-intro: Facilitate a mandatory two-hour training for all SMEs participating in an assessment round. The training should explain how to review the assessment against the required proficiency levels.
+intro: Hold a mandatory SME training for each type of assessment, showing them how to review the responses against the required specialized experience (e.g. minimum  proficiency level set during job analysis)
```

Comment:
i edited this to be more clear it is not just leveraging proficiencies but assessing against that min bar.

Reply:
I updated this edit to make it shorter. Also, SMEQA is always about competencies/proficiencies - do we need to say specialized experience in the intro?